web/src/lib/components/pages/profile/profile.tsx

```diff
@@ -178,7 +132,7 @@ export default class Profile extends Component<Props, State> {
           ref={select => {
             this.profileAccent = select as HTMLSelectElement;
           }}>
-          {accentOptions}
+          {this.renderOptionsFor(ACCENTS, accent)}
         </select>
```

Comment:
i think we should extract these entire `<select>`s (including their labels) into a component, to which we pass the data needed (id, tab index, isModified, options, currentOption).

that component could be local to this file, doesn't necessarily have to be in a separate file IMO

Reply:
The problem is that React <16 can't return arrays in render. Otherwise I'd be on your side on moving it to its own component.

Reply:
ugh. that's annoying. though couldn't we just wrap them in another div then?

but i see that's not a trivial extraction then, so let's maybe attempt this in another PR